src/python/visclaw/data.py

```diff
@@ -25,7 +25,7 @@ class ClawPlotData(clawdata.ClawData):
     """
 
     # ========== Initialization routine ======================================
-    def __init__(self, controller=None):
+    def __init__(self, controller=None, num_dim=2):
```

Comment:
I thought by specifying a "default" value for num_dim, all the existing init calls would still work?  My python experience is pretty limited... so I'm probably missing something.  Looking at the rest of the comments, this should be possible without adding the attribute.